staging/src/k8s.io/apiserver/pkg/server/options/serving_test.go

```diff
@@ -387,19 +401,22 @@ func TestServerRunWithSNI(t *testing.T) {
 		},
 	}
 
-	tempDir, err := ioutil.TempDir("", "")
-	if err != nil {
-		t.Fatal(err)
+	specToName := func(spec TestCertSpec) string {
+		return spec.host + "+" + strings.Join(spec.names, ",") + "+" + strings.Join(spec.ips, ",")
```

Comment:
Would it also be worth including `host=`, `names=`, and `ips=` in the file name?